### ISSUE TITLE
Uncaught reference error: iced is not defined

### ISSUE THREAD

**Issue by mrft** [maxtaco/icsify#3]
So I have my simple html file that includes <pre>&lt;script src="browserified/main.js"></script></pre>

That file has been created using
<code>
browserify js/main.iced -t icsify -d -o browserified/main.js
</code>

At the first defer statement, the browser console shows: "Uncaught reference error: iced is not defined"

What should I do about it? Or should you fix something in this module?
(I read something about the --runtime inline option on the ics compiler, but I don't know - browserify n00b -  how to enable that, if possible, on the icsify module).

**Comment by maxtaco** [comment]
We use a two-step output process for browserification:

```
iced -I browserify -o js/main.js js/main.iced
browserify js/main.js -t icsify -d -o browserified/main.js
```

**Comment by maxtaco** [comment]
Can you send your `js/main.iced` or a simplified version of it so I can take a look?

**Comment by mrft** [comment]
Hmm,

My script was something like:
<pre>
console.log( 'Browserify has done the job...' )
alert( "It is working" )

await setTimeout( defer(), 1000 )
console.log( "await - defer() is working..." )
</pre>

but for some reason, the first time it didn't complain about missing iced-runtime. So I started over with an empty project.

Then it started complaining about 'missing iced-runtime', so it was pretty easy to find out that I also had to <pre>npm install iced-runtime</pre>.

Now everything is working as expected.

Sorry for bothering you...
